zanata-war/src/main/java/org/zanata/action/ProjectHome.java

```diff
@@ -286,8 +286,6 @@ public void setSelectedProjectType(String selectedProjectType) {
                 && !selectedProjectType.equals("null")) {
             ProjectType projectType = ProjectType.valueOf(selectedProjectType);
             getInstance().setDefaultProjectType(projectType);
-        } else {
-            getInstance().setDefaultProjectType(null);
```

Comment:
This looks like it doesn't have much to do with navigation. Why did we remove these lines?